ouroboros-consensus-cardano/src/unstable-shelley-testlib/Test/Consensus/Shelley/Examples.hs

```diff
@@ -228,6 +241,11 @@ fromShelleyLedgerExamples
       ExtLedgerState
         ledgerState
         (genesisHeaderState chainDepState)
+    perasCert =
+      PerasCert
+        { pcCertRound = PerasRoundNo 10
+        , pcCertBoostedBlock = blockPoint blkWithoutCert
```

Comment:
We probably want to have a different boosted block, but this one was convenient for now.